### PR TITLE
improved alternate link placement on login and signup page

### PR DESCRIPTION
**Pull request by salmanmanekia**
#### Summary

Currently, the alternate link anchors away from the login and signup form if the brand description or brand image occupies alot of space. This PR fixes it so the alternate link remains in its place

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-45803

#### Screenshots

|  before  |  after  |
|----|----|
| login-before | login-after |

#### Release Note
```release-note
NONE
```

### CODE REVIEW

File: components/login/login.tsx [hunk 1]
```diff
@@ -736,82 +736,85 @@ const Login = ({onCustomizeHeader}: LoginProps) => {
...
-                                    containerClassName='login-body-card-form-input'
-                                    type='text'
-                                    inputSize={SIZE.LARGE}
-                                    value={loginId}
-                                    onChange={handleInputOnChange}
-                                    hasError={hasError}
-                                    placeholder={getInputPlaceholder()}
-                                    disabled={isWaiting}
-                                    autoFocus={true}
+                <div className='login-body-action'>
```
Comment (julmondragon) [inline components/login/login.tsx:739]: When custom branding is applied the alignment between `login-body-message` and `login-body-card` is also changing:

Before:
Screen Shot 2022-09-07 at 10 13 32 AM

After:
Screen Shot 2022-09-07 at 10 15 10 AM

Maybe compensating `alternate-link`'s height in the `login-body-message` side can help keep the expected alignment, but take into account that `alternate-link` is not always present (ie. No open server enabled)

File: components/login/login.scss [hunk 2]
```diff
@@ -8,8 +8,15 @@
...
 
+    .login-body-action {
+        display: flex;
+        flex-direction: column;
+        align-items: flex-end;
+    }
+
     .login-body-alternate-link {
         margin-bottom: 18px;
+        margin-left: 60px;
```
[the remark this answers is not in the document]
Comment (salmanmanekia) [reply components/login/login.scss]: Nice catch. It remained there by mistake as I was testing something. I have removed it now from login and signup scss.

File: components/login/login.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (salmanmanekia) [reply components/login/login.tsx:739]: Looked thin. Done as you suggested and it looks better now.

Comment (salmanmanekia) [reply components/login/login.tsx:739]: Added `margin-top: 40 px` to `login-body-message`. 
Didn't had to add conditional css as `login-body-message` is invisible on smaller screens where `alternate-link` moves away to top right corner or is invisible.

**Review by julmondragon — APPROVED** [review]
LGTM!

**Comment by salmanmanekia** [comment]
I missed a few things, thanks @julmondragon for the commit. @stevemudie am I good to merge this PR ?

**Comment by matthewbirtch** [comment]
I was waiting for the items from the review @julmondragon did to be resolved before reviewing. I'll try to do a review today if I can

**Review by matthewbirtch — APPROVED** [review]
Looks good to me!

**Pull request merged**